src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final int EXPECTED_VALUES_SIZE = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int [] fromFileData = readMethod(fromFileName);
+        String result = getResult(fromFileData);
+        createNewFile(result, toFileName);
+
+    }
```

Comment:
The method name `getStatistic` does not accurately describe the actions being performed within the method. A name like `generateReport` would be more descriptive, as the method reads data, processes it, and writes the result to a file.